src/Utils/hiopOptions.cpp

```diff
@@ -160,13 +160,12 @@ void hiopOptions::load_from_file(const char* filename)
   }
 
   ifstream input( filename );
-
   if(input.fail()) {
     if(strcmp(default_filename, filename)) {
       log_printf(hovWarning,
                  "Failed to read option file '%s'. Hiop will use default options.\n",
                  filename);
-      return;
+//      return;
```

Comment:
@kswirydo, why is return statement commented out here? I would think if one should exit the function.

Reply:
just uncomment it. THe answer is: I dont know (nor remember)